doc/source/whatsnew/v1.1.4.rst

```diff
@@ -20,6 +20,7 @@ Fixed regressions
 - Fixed regression in :class:`RollingGroupby` with ``sort=False`` not being respected (:issue:`36889`)
 - Fixed regression in :meth:`Series.astype` converting ``None`` to ``"nan"`` when casting to string (:issue:`36904`)
 - Fixed regression in :class:`RollingGroupby` causing a segmentation fault with Index of dtype object (:issue:`36727`)
+- Fixed regression in :meth:`Resample.apply` raised ``AttributeError`` when input was a :class:`DataFrame` and only a :class:`Series` was evaluated (:issue:`36951`)
```

Comment:
i don't think will render for Resample you need to fully qualified method (look around for examples in this whatsnew or others)

Reply:
Thanks found an example